conda/recipes/rapids_dask_dependency/meta.yaml

```
@@ -0,0 +1,29 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+package:
+- name: rapids_dask_dependency
+  version: 23.10.00.0
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - dask >=2023.7.1
+    - dask-core >=2023.7.1
+    - distributed >=2023.7.1
```

As `dask` pulls in the others (and tightly pins them), we can just pin `dask`

If we do want to list these all for completeness, would suggest templating the versions to make things a bit more DRY

```suggestion
    - dask >=2023.7.1
```

---

I think keeping them separate still makes sense. Imagine the previous case with Distributed that required us to release a patch release `distributed=7.3.2.1`, whereas `dask=7.3.2`, if we have such a situation in future again this would prevent us from pinning to different patch versions.